website/src/components/AddDevice.tsx

```diff
@@ -100,6 +100,7 @@ export const AddDevice = observer(class AddDevice extends React.Component<Props>
         PrivateKey = ${privateKey}
         Address = ${device.address}
         ${ 0 < dnsInfo.length && `DNS = ${ dnsInfo.join(", ") }` }
+        MTU = ${info.mtu}
```

Comment:
This is the main component that fixes the issue.

Reply:
As proposed please replace with the client configuration item `clientConfig.mtu`.
If this item is not set, the `MTU = ...`-line should not be printed into the client configuration file.